### PR TITLE
Adding RPM spec file instructions to build on SUSE SLES11 / SLES12

### PR DESCRIPTION
**Pull request by sbernhard**

### CODE REVIEW

File: packages/pulp/pulp.spec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 %{!?python_sitearch: %global python_sitearch %(%{__python} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(1))")}
 
 # don't use <= 6 here, since %{?rhel} is empty on fedora, and 0 is <= 6
-%if 0%{?rhel} == 5 || 0%{?rhel} == 6
+%if 0%{?rhel} == 5 || 0%{?rhel} == 6 || 0%{?suse_version}
```
Comment (ekohl) [inline packages/pulp/pulp.spec:5]: Does that mean only the pulp consumer is built on SUSE?

Comment (sbernhard) [reply packages/pulp/pulp.spec:5]: Only the client part is required to use the katello-agent on SUSE.

**Comment by pcreech** [comment]
@sbernhard Thanks for the PR!  I'm in training this week, but I should get a moment to look it over in the next couple days

**Comment by pcreech** [comment]
I can't speak to the functionality for suse, as I don't have the infrastructure to test such, but these changes do not break building for fedora or el, so I'm inclined to accept them

**Review by pcreech — APPROVED** [review]

**Pull request merged**